corehq/apps/app_manager/static/app_manager/js/details/case_claim.js

```diff
@@ -343,35 +343,35 @@ hqDefine("app_manager/js/details/case_claim", function () {
         self.defaultProperties = ko.observableArray();
 
         if (searchProperties.length > 0) {
-            for (var i = 0; i < searchProperties.length; i++) {
-                // searchProperties is a list of CaseSearchProperty objects
+            // searchProperties is a list of CaseSearchProperty objects
+            _.each(searchProperties, function (searchProperty) {
```

Comment:
yay, seeing inconsistent iteration styles bugs me. This file so does not look like a single person wrote it.

This could be a `_.map`, couldn't it?

Reply:
Yeah probably - I had fiddled with a couple ways of doing this, ended up punting.  I just switched it to a map and moved the `self.searchProperties` initialization below, lemme know what you think.

Reply:
did you push? I don't see anything new.

Reply:
I see it now, I like it!